### DISCUSSION TITLE
Custom instance URL

### DISCUSSION THREAD

**Discussion by cs-vahan** [arnoudkooi/SN-Utils#375] [Q&A] [answered]
Currently only *.service-now.com instance FQDNs seem to be supported. Is there a way to add custom instance names? For example: prod.mycompanyname.com ?

Thanks

**Comment by arnoudkooi** [comment, marked as answer]
All non service-now.com instances require the on prem version
https://chrome.google.com/webstore/detail/sn-utils-onprem/lfabkiipmidkmhplochgpbaeekjjfbch?hl=en-GB&authuser=0

**Reply by cs-vahan** [reply]
Thanks, not sure how i missed that!
